### ISSUE TITLE
pyparsing issue 

### ISSUE THREAD

**Issue by guilleaf** [wolverton-research-group/qmpy#19]
Hello, 

There is an issue trying to build qmpy from the git repo.

Processing dependencies for qmpy==0.4.9a0
Searching for pyparsing<=1.9.9
Reading https://pypi.python.org/simple/pyparsing/
Downloading https://pypi.python.org/packages/6f/2c/47457771c02a8ff0f302b695e094ec309e30452232bd79198ee94fda689f/pyparsing-1.5.7.tar.gz#md5=9be0fcdcc595199c646ab317c1d9a709
Best match: pyparsing 1.5.7
Processing pyparsing-1.5.7.tar.gz
Writing /tmp/easy_install-XIv02R/pyparsing-1.5.7/setup.cfg
Running pyparsing-1.5.7/setup.py -q bdist_egg --dist-dir /tmp/easy_install-XIv02R/pyparsing-1.5.7/egg-dist-tmp-7hJ2IA
zip_safe flag not set; analyzing archive contents...
pyparsing: module MAY be using inspect.stack
creating /home/gufranco/.local/lib/python2.7/site-packages/pyparsing-1.5.7-py2.7.egg
Extracting pyparsing-1.5.7-py2.7.egg to /home/gufranco/.local/lib/python2.7/site-packages
Adding pyparsing 1.5.7 to easy-install.pth file

Installed /home/gufranco/.local/lib/python2.7/site-packages/pyparsing-1.5.7-py2.7.egg
error: pyparsing 1.5.7 is installed but pyparsing>=2.0.1 is required by set(['PuLP'])

You can see the same issue reproduce on travis.

Basically setup.py ask for pyparsing being <= 1.9.9 and PuLP ask for pyparsing being >= 2.0.1

Why qmpy ask for an older version of pyparsing?

I contour the issue changing setup.py
What could be the side effect of using a more recent version of pyparsing.

Thank you very much, 
And thank you for your nice python package.

**Comment by hegdevinayi** [comment]
@guilleaf sorry for the late response but if you haven't tried building qmpy recently: it no longer requires an old version of pyparsing (fixed in d458d2f145f0fbaac0afb6b84b0a517922795517). Closing
